sdk/nodejs/yaml/yaml.ts

```diff
@@ -3069,6 +3069,14 @@ export interface ConfigOpts {
     if (resourcePrefix !== undefined) {
         id = pulumi.concat(resourcePrefix, "-", id);
     }
+
+    if (gvk === "v1/Secret") {
```

Comment:
Should we merge this with the switch case for `v1/Secret` in the block that follows instead?

Reply:
I kept it separate because it's nontrivial to update the mustache template generating this file otherwise. Otherwise, yes. :)

Reply:
Ah missed that. Gotcha. Looks good! 

Optional - perhaps factor this out to a function so it doesn't distract from the rest of the parsing stuff?

Reply:
I guess we now need to do this for all the other languages.

Reply:
> I guess we now need to do this for all the other languages.

Yep, just fighting with typing on C#, but nearly done. Pushed Python and Go.